hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/HazelcastSqlToRelConverter.java

```diff
@@ -113,14 +112,13 @@ public final class HazelcastSqlToRelConverter extends SqlToRelConverter {
     private final Set<SqlNode> callSet = Collections.newSetFromMap(new IdentityHashMap<>());
 
     public HazelcastSqlToRelConverter(
-            RelOptTable.ViewExpander viewExpander,
             SqlValidator validator,
             Prepare.CatalogReader catalogReader,
             RelOptCluster cluster,
             SqlRexConvertletTable convertletTable,
             Config config
     ) {
-        super(viewExpander, validator, catalogReader, cluster, convertletTable, config);
+        super(null, validator, catalogReader, cluster, convertletTable, config);
```

Comment:
why not just remove this argument, instead of passing null?

Reply:
The superclass ```SqlToRelConverter``` is from Calcite, it is not in our codebase.